src/components/index/sections/SupportUsSection.tsx

```diff
@@ -10,42 +10,45 @@ const useStyles = makeStyles((theme) =>
   createStyles({
     container: {
       backgroundColor: '#eeeeee',
-      paddingTop: theme.spacing(10),
+      padding: theme.spacing(6, 0),
+      [theme.breakpoints.up('md')]: {
+        padding: theme.spacing(6, 0, 0),
+      },
     },
-    heading: {
-      marginBottom: theme.spacing(4),
-      color: theme.palette.primary.dark,
-      fontWeight: 500,
+    supportImageWrapper: {
+      textAlign: 'center',
     },
-    icon: {
-      fontSize: theme.typography.pxToRem(80),
-      fill: theme.palette.primary.main,
-      padding: theme.spacing(1),
+    supportImage: {
+      maxWidth: '60%',
+      [theme.breakpoints.down('sm')]: {
+        marginBottom: theme.spacing(3),
+      },
     },
     supportOptionsWrapper: {
-      textAlign: 'center',
-      [theme.breakpoints.up('md')]: {
-        textAlign: 'left',
+      [theme.breakpoints.down('sm')]: {
+        textAlign: 'center',
       },
     },
-    supportImage: {
-      width: '40%',
-      height: '100%',
+    heading: {
+      marginBottom: theme.spacing(3),
+      color: theme.palette.primary.dark,
+      fontWeight: 500,
     },
     clickHereButton: {
       color: theme.palette.common.white,
       backgroundColor: theme.palette.primary.main,
       border: `2px solid ${theme.palette.primary.main}`,
       borderRadius: theme.spacing(3),
-      padding: theme.spacing(1, 5),
-      margin: theme.spacing(4, 0),
-      fontWeight: 500,
+      width: theme.spacing(25),
```

Comment:
Fixed height and width will be troublesome at some point, we better restrict those with `minWidth` and `minHeight` if really necessary.